hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1302,6 +1303,24 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 			controlPlaneNamespace.Labels["openshift.io/cluster-monitoring"] = "true"
 		}
 
+		// Set a label so we can identify the control plane namespaces.
+		controlPlaneNamespace.Labels[ControlPlaneNamespaceLabelKey] = "true"
```

Comment:
We already set a label that identifies the control plane namespace:
https://github.com/openshift/hypershift/blob/f70206108efa2e208c4733f2f819ce639e1a8a75/hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go#L1281